### PR TITLE
2.3.2 release notes

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: content/reference/release-notes/portworx.md [hunk 1]
```diff
@@ -6,6 +6,37 @@ keywords: portworx, release notes
...
+| PWX-10380 | SharedV4 volumes are now enabled when SELinux is enabled on Portworx nodes. If you expect SELinux labels to be propagated from an NFS client to a server, set the `ExportOptions` on a volume to `security_label`. You can use the following command to update the volume option: `pxctl volume update --export_options security_label <vol-name>` |
+
+### Fixes
+
+The following issues have been fixed:
+
+|**Issue Number**|**Issue Description**|
+|----|----|
+| PWX-10366 | Portworx did not delete node region or zone values when instructed. <br/><br/>**User Impact:** Portworx continued to show deleted node and region zone labels after users deleted them. This issue persisted over restarts. <br/><br/>**Resolution:** Portworx now properly deletes these labels and replaces them with the `default` value. |
+| PWX-10381 | Do not transfer license of an offline storage less node to any available nodes when running on an on-prem cluster. This feature is enabled only when deployed in cloud. |
```
Comment (nathan-px) [inline content/reference/release-notes/portworx.md]: @adityadani Should we remove this?

File: content/reference/release-notes/portworx.md [hunk 2]
```diff
@@ -6,6 +6,37 @@ keywords: portworx, release notes
...
+| PWX-10455 | A failure during a volume create operation can result in a partially formatted volume. A subsequent attach on this volume will retry the formatting operation. In the case of xfs volumes, this formatting operation can fail if the new operation finds an xfs signature on the volume from the previous incomplete operation. <br/><br/>**User Impact:** Partially formatted xfs volumes could not be attached.<br/><br/>**Resolution:** Portworx now uses the force flag when retrying the format operation. |
+| PWX-10657 | The etcdv3 client Portworx uses currently contains the following critical bug: https://github.com/etcd-io/etcd/pull/10911. When connected to a secure etcd cluster, if the first endpoint goes offline, the etcd client does not failover and fails to create a new connection. <br/><br/>**User Impact:** Portworx restarts and does not reconnect to the etcd cluster. <br/><br/>**Resolution:** After restarting, Portworx now reshuffles the list of endpoints so that the etcd client reconnects to the cluster. |
+| PWX-10383 | Storageless nodes will not mark peer storageless nodes offline as they do not contribute to quorum. |
```
Comment (nathan-px) [inline content/reference/release-notes/portworx.md]: @adityadani and this one?

File: content/reference/release-notes/portworx.md [hunk 3]
```diff
@@ -6,6 +6,44 @@ keywords: portworx, release notes
...
+December 18, 2019
+
+### Improvements
+
+Portworx has upgraded or enhanced functionality in the following areas:
+
+|**Improvement Number**|**Improvement Description**|
+|----|----|
+| PWX-10095 | Portworx now restricts access to SharedV4 volumes to nodes requesting a mount on that volume. |
+| PWX-10499 | The `storage pool expansion` operation now supports the `auto` option. |
```
Comment (adityadani) [inline content/reference/release-notes/portworx.md]: @vilasd We don't need to add this to release notes as we are hiding the `pool expand` command itself right ?
2.4.1 can have a separate improvement release note that talks about the new `pool expand` command

File: content/reference/release-notes/portworx.md (hunk 2 above)
Comment (adityadani) [reply content/reference/release-notes/portworx.md]: We don't need this one.

File: content/reference/release-notes/portworx.md [hunk 4]
```diff
@@ -6,6 +6,44 @@ keywords: portworx, release notes
...
+| PWX-10468 | On nodes which were also volume servers and had attached SharedV4 volumes, Portworx did not restart Application pods when it entered maintenance mode or was decomissioned. <br/><br/>**User Impact:** Users experienced I/O errors caused by missing application pods. <br/><br/>**Resolution:**  Portworx nodes now delete the application pods to recover the application. |
+| PWX-10575 | On AKS clusters with VM scale sets, if a Portworx node with cloud drives failed to bootstrap, detach operations also failed.<br/><br/>**User Impact:** Cloud drives remained attached to non-existent Portworx nodes.<br/><br/>**Resolution:** Portworx now properly detaches cloud drives if it fails to bootstrap a node. |
+| PWX-10525 | Portworx frequently queried etcd to retrieve the storage spec and check storage pool status and pending drive operations. <br/><br/>**User Impact:** These frequent queries placed unnecessary load on etcd, resulting in excessive resource usage and decreased performance. <br/><br/>**Resolution:** This fix limits the periodic calls and makes them only when necessary: on a version update. |
```
Comment (adityadani) [inline content/reference/release-notes/portworx.md]: This bug was introduced in 2.3.0. Can we mention that?
Customers who are using older releases, should not panic and update to this release as most of our customers are very conservative on how we use etcd.

File: content/reference/release-notes/portworx.md (hunk 3 above)
Comment (vilasd) [reply content/reference/release-notes/portworx.md]: We need to release note

**Pull request merged**